scalikejdbc-core/src/main/scala/scalikejdbc/ConnectionPoolSettings.scala

```diff
@@ -10,5 +10,6 @@ case class ConnectionPoolSettings(
   validationQuery: String = null,
   connectionPoolFactoryName: String = null,
   driverName: String = null,
-  warmUpTime: Long = 100L)
+  warmUpTime: Long = 100L,
+  timeZone: String = null)
```

Comment:
Why not using `Option[String]` here? Though I don't know why other fields are also nullable. @seratch